include/camp/number.hpp

```diff
@@ -120,6 +120,23 @@ struct not_ {
   using type = typename if_s<T, false_type, true_type>::type;
 };
 
+
+
+template <idx_t N, typename IdxSeg>
+struct seq_at;
+
+template <idx_t N, idx_t Idx0, idx_t ... IdxRest>
+struct seq_at<N, camp::idx_seq<Idx0, IdxRest...>>{
```

Comment:
This should not have been necessary, I'll put up an issue for this and fix idx_seq so we can use the regular at_v on it and get back a num<n>.  I forgot that I dropped the camp::list<num<n>> backing for these. =/

Reply:
Sounds good. at_v was the first thing I tried, but it wasn't clear to me how to extend it.

Reply:
It's harder than it should be, since an int_seq isn't a camp::list, so I think I'll add a converting overload that gives out a camp::integral_constant<IntT, Val> at the given offset.  Alternately we can keep this and have a helper to convert an {int,idx}_seq into a camp::list so at works.